From 737f01784e0cec924ef97053f9ba17878f352605 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Mon, 19 Aug 2019 21:33:08 +0900
Subject: [PATCH] We don't use menu anymore

> The menu system is deprecated in favor of the FreeDesktop menu standard.
see https://www.debian.org/doc/debian-policy/upgrading-checklist.html#version-3-9-8
---
 lib/debian/menu.ex | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 lib/debian/menu.ex

diff --git a/lib/debian/menu.ex b/lib/debian/menu.ex
deleted file mode 100644
index 6f02d89..0000000
--- a/lib/debian/menu.ex
+++ /dev/null
@@ -1,2 +0,0 @@
-?package(#PACKAGE#):needs="X11|text|vc|wm" section="Applications/see-menu-manual"\
-  title="#PACKAGE#" command="/usr/bin/#PACKAGE#"
-- 
2.23.0.rc1

